lalrpop-test/src/expr.lalrpop

```diff
@@ -11,6 +11,7 @@ extern {
         "*" => Tok::Times,
         "/" => Tok::Div,
         Num => Tok::Num(<i32>),
+        Fraction => Tok::Fraction(<i32>, <i32>),
```

Comment:
Can you add a comment like `// Regression test for #179` or something so that we remember that?